austation/code/_globalvars/misc.dm

```diff
@@ -1 +1,4 @@
 GLOBAL_LIST_INIT(ringlist, list(RING_DISABLED_NAME, RING_CASUAL_NAME, RING_ENGAGEMENT_NAME, RING_WEDDING_NAME, RING_AUSTRALIUM_NAME))
+
+GLOBAL_VAR_INIT(total_beacons, 0)
+GLOBAL_VAR_INIT(max_beacons, 100)
```

Comment:
you're setting a global variable for a constant value? use a define for constant values like that, especially simple numbers.

Reply:
removed global limit